indico/htdocs/sass/modules/_abstracts.scss

```diff
@@ -16,3 +16,7 @@
  */
 
 @import 'abstracts/abstract_page';
+
+.submitter-column {
```

Comment:
I don't know if we want to mix imports and CSS rules. Maybe you could create an `abstract/_abstract_list.scss` file? Not a strong opinion.